packages/workflow/src/utils.ts

```diff
@@ -127,3 +128,14 @@ export function assert<T>(condition: T, msg?: string): asserts condition {
 		throw error;
 	}
 }
+
+export const isFrontend = () => typeof process === 'undefined';
```

Comment:
do we need to evaluate this everytime? or do you think we could just assign this once?
```suggestion
export const isFrontend = typeof process === 'undefined';
```